doc/tutorials/highgui/trackbar/trackbar.markdown

```diff
@@ -88,10 +88,10 @@ Explanation
 
 We only analyze the code that is related to Trackbar:
 
--#  First, we load 02 images, which are going to be blended.
+-#  First, we load two images, which are going to be blended.
     @code{.cpp}
-    src1 = imread("../../images/LinuxLogo.jpg");
-    src2 = imread("../../images/WindowsLogo.jpg");
+    src1 = imread("../data/LinuxLogo.jpg");
+    src2 = imread("../data/WindowsLogo.jpg");
     @endcode
```

Comment:
I just saw some [discussion](https://github.com/opencv/opencv/issues/6938) of @mshabunin that he wants to avoid adding `@code` commands and replace them completely by parts of code snippets through the `@include` command. Shouldn't we adapt this on the spot?